src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -40,6 +40,9 @@
         "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/var-dumper": "~2.6|~3.0.0"
     },
+    "conflict": {
+        "symfony/config": "<2.7"
```

Comment:
@fabpot Do you prefer a PR with tests "all green", or shall I keep changes to a minimum at the cost of having dep= tests pass only once the change has been merged up to the master branch?

I think this is a special case here because changes are made across components that are then tested independently.